From 8b6be4c65eda396e0c42182f4ebf2794c62b442c Mon Sep 17 00:00:00 2001
From: Josef Bacik <josef@redhat.com>
Date: Tue, 2 Feb 2010 09:51:15 -0500
Subject: [PATCH] Btrfs-progs: make btrfsctl return values like everybody else

This patch makes btrfsctl return 0 for success and 1 for failure, which is the
behavior of all other userspace applications.  Thanks,

Signed-off-by: Josef Bacik <josef@redhat.com>
---
 btrfsctl.c |    6 ++----
 1 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/btrfsctl.c b/btrfsctl.c
index 66c4e89..73d9c30 100644
--- a/btrfsctl.c
+++ b/btrfsctl.c
@@ -231,9 +231,7 @@ int main(int ac, char **av)
 		printf("ioctl failed with error %d\n", ret);
 	}
 	printf("%s\n", BTRFS_BUILD_VERSION);
-	if (ret)
-		exit(0);
-	else
-		exit(1);
+
+	return ret ? 1 : 0;
 }
 
-- 
1.6.6

